frontend/src/metabase/visualizations/components/ChoroplethMap.jsx

```diff
@@ -146,11 +146,14 @@ export default class ChoroplethMap extends Component {
     let { geoJson, minimalBounds } = this.state;
 
     // special case builtin maps to use legacy choropleth map
-    let projection;
+    let projection, projectionFrame;
+    // projectionFrame is the lng/lat of the top left and bottom right corners
     if (settings["map.region"] === "us_states") {
       projection = d3.geo.albersUsa();
+      projectionFrame = [[-135.0, 46.6], [-69.1, 21.7]];
     } else if (settings["map.region"] === "world_countries") {
       projection = d3.geo.mercator();
+      projectionFrame = [[-170, 78], [180, -60]];
```

Comment:
This frame was hand-crafted and crops of Antarctica, part of Greenland, and a few other tiny parts. It should be easy to adjust if we want to update the framing.